daffodil-test/src/test/resources/org/apache/daffodil/section17/calc_value_properties/computedLengthFields.tdml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<testSuite suiteName="Computing Length Fields"
+           xmlns="http://www.ibm.com/xmlns/dfdl/testData"
```

Comment:
Should we make this change in the project layout template? Should *.dfdl.xsd not define the xmlns:xs prefix and should *.tdml files not define the xmlns:tdml prefix, and instead we should just set xmlns appropriately?

Reply:
I'm not quite ready to go there yet. 

You also need to use tdml:defineSchema options elementFormDefault 'unqualified' and useDefaultNamespace 'false', to get this to work. Those *should* both be the default. But for historic backward compatibility, they are not.